README.md

```diff
@@ -74,22 +74,7 @@ upstream-community-operators   federationv2,svcat,metering,etcd,prometheus,autom
 ```
 **_Note_**: Please do not install [upstream-community-operators] and [community-operators] `OperatorSources` on the same cluster. The rule of thumb is to install [community-operators] on OpenShift clusters and [upstream-community-operators] on upstream Kubernetes clusters.
 
-Now if you want to install the `descheduler` and `jaeger` operators, create a `CatalogSourceConfig` CR as shown below:
-```
-apiVersion: operators.coreos.com/v1
-kind: CatalogSourceConfig
-metadata:
-  name: installed-upstream-community-operators
-  namespace: marketplace
-spec:
-  targetNamespace: local-operators
-  packages: descheduler,jaeger
-  csDisplayName: "Upstream Community Operators"
-  csPublisher: "Red Hat"
-```
-Note, that in the example above, `local-operators` is a namespace that OLM is watching. Deployment of this CR will cause a `CatalogSource` called `installed-upstream-community-operators` to be created in the `local-operators` namespace. This can be confirmed by `oc get catalogsource installed-upstream-community-operators -n local-operators`. Note that you can reuse the same `CatalogSourceConfig` for adding more operators.
-
-Now you can create OLM [`Subscriptions`](https://github.com/operator-framework/operator-lifecycle-manager/tree/274df58592c2ffd1d8ea56156c73c7746f57efc0#discovery-catalogs-and-automated-upgrades) for `desheduler` and `jaeger`.
+Now if you want to install the `descheduler` and `jaeger` operators, create OLM [`Subscriptions`](https://github.com/operator-framework/operator-lifecycle-manager/tree/274df58592c2ffd1d8ea56156c73c7746f57efc0#discovery-catalogs-and-automated-upgrades) for `desheduler` and `jaeger`.
```

Comment:
- Please update the section about `CatalogSourceConfig`. `CatalogSourceConfig` is used to create `CatalogSource` subsets from an `OperatorSource`. 

Should we call out that `OperatorSources` result in the creation of a `CatalogSource` that can then be used for creating OLM `Subscriptions` in the `OperatorSource` section?

Reply:
Yes we probably should

Reply:
For your first suggestion, should that not go in #197 ? Or I can change it to be a `CatalogSourceConfig` is used to create combination of `CatalogSources` because there is no subsets restriction yet

Reply:
Sure. It creates a CatalogSource from one or more `OperatorSources` not a combination of `CatalogSources`.